src/lib.rs

```diff
@@ -246,9 +246,7 @@ pub use result::{IonError, IonResult};
 /// Re-exports of third party dependencies that are part of our public API.
 ///
 /// See also: <https://github.com/amazon-ion/ion-rust/issues/302>
-pub mod external {
-    pub use bigdecimal;
-}
+pub mod external {}
```

Comment:
🗺️ I think we still need this for dependencies behind opt-in feature gates.

Reply:
(Question) Can we remove this module as it doesn't have any dependencies at the moment? Later when we need re-exports for the opt-in features, we can create the module for it.
Having this in crate docs with no dependencies in it might be confusing, so we should either add a comment or an issue link stating why we have kept it or indicating why this is empty at the moment.

Reply:
Great point, I'll take it out.